flashinfer/aot.py

```diff
@@ -371,24 +338,33 @@ def gen_all_modules(
             jit_specs.append(gen_gemm_sm100_module())
 
     if add_comm:
-        from .comm import gen_trtllm_comm_module, gen_vllm_comm_module
+        try:
+            from .comm.nvshmem import gen_nvshmem_module
 
-        if has_sm100:
-            jit_specs.append(gen_trtllm_comm_module())
-        jit_specs.append(gen_vllm_comm_module())
+            jit_specs.append(gen_nvshmem_module())
+        except ImportError:
+            pass
```

Comment:
I guess we should raise here. Because `if add_comm` explicitly request for comm.